packages/backend/src/core/activity/processors/getBatchSizeFromCallsPerMinute.ts

```diff
@@ -0,0 +1,3 @@
+export function getBatchSizeFromCallsPerMinute(callsPerMinute: number): number {
```

Comment:
This function make me very suspicious. If `callsPerMinute` is less than 60 it returns 0, but the minimum should be 1. I assume the idea was to make it so a batch takes roughly 1s, but Maybe make this explicit with another parameter, e.g. `batchDurationSeconds`.